### ISSUE TITLE
Jobs column un-readable when long ID's are using in ID column in recurring jobs Dashboard

### ISSUE THREAD

**Issue by john-luke-laue** [HangfireIO/Hangfire#1343]
Sometimes, the jobs column will be unreadable.
This is because it does not have a "min-width" css class like all the other columns.

Is there any way I can customize this myself? 
Or would you accept a pull-request where I add a "min-width" css class to the Jobs column?

![image](https://user-images.githubusercontent.com/1404286/52075198-8cb02700-2540-11e9-9e07-bd82791b4dcc.png)

**Comment by ckpearson** [comment]
:+1: We're bumping into this issue too, and confirm the detail re: min-width css class.

Would be good to get the css class applied to the column, and maybe even enabling the entire UI to expand outwards a little more, as we have some jobs with fairly long IDs.

**Comment by EricsG95** [comment]
Yes I am facing similar issues and unable to update css or whatsoever.

![image](https://user-images.githubusercontent.com/10571842/71954821-79faa380-3221-11ea-9111-3000a65676c4.png)

It will be great if this part of css be updated with "overflow: auto".
![image](https://user-images.githubusercontent.com/10571842/71954898-b75f3100-3221-11ea-94bd-6bf3316267d3.png)

In cases of longer results, information will be wrapped.
